<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElementWidget</class>
 <widget class="QWidget" name="ElementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>101</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>0.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="theIcon">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>9</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/ElementWidget/Resources/male.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="theName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="theSquareLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/ElementWidget/Resources/square.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="theBirthdate">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Birth Date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>theSquareLabel</zorder>
  <zorder>theIcon</zorder>
  <zorder>theName</zorder>
  <zorder>theBirthdate</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
